automation-ui/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.yas</groupId>
+		<artifactId>yas</artifactId>
+		<version>${revision}</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+	<artifactId>automation-ui</artifactId>
+	<version>${revision}</version>
+	<packaging>pom</packaging>
+	<name>automation-ui</name>
+	<description>YAS automation-ui test</description>
+	<modules>
+		<module>storefront</module>
+	</modules>
+	<properties>
```

Comment:
move the sonar properties to storefront. the sonar.projectKey value should be nashtech-garage_yas-automation-ui-storefront